src/main/java/tests/FileReaderWriterTest.java

```diff
@@ -0,0 +1,29 @@
+package tests;
+
+import core.basesyntax.FileReader;
+import core.basesyntax.FileReaderImpl;
+import core.basesyntax.FileWriter;
+import core.basesyntax.FileWriterImpl;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class FileReaderWriterTest {
+    @Test
+    public void testFileReaderWriter() throws IOException {
+        String filePath = "testReport.csv";
```

Comment:
Avoid using hardcoded file paths in tests. It's better to use a temporary file or a mock to ensure that the tests do not affect the local file system and are more reliable. Consider using JUnit's `@TempDir` to create a temporary directory for the files.